torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -1367,6 +1374,10 @@ Stmt* TensorExprKernel::generateStmt(BackendType backendType) {
 
     // vectorize inner loops.
     for (For* loop : innerLoops) {
+      if (NodeFinder<ReduceOp>::find(loop).size() > 0) {
```

Comment:
Is it needed given the `allowVectorization` check above?

Reply:
Oops, good catch! Done.